src/patient-registration/patient-registration.resource.tsx

```diff
@@ -12,6 +12,17 @@ export function savePatient(abortController: AbortController, patient: Patient)
   });
 }
 
+export function saveRelationships(abortController: AbortController, relationships: tempRelationship) {
```

Comment:
You should type the return value from this function.

Reply:
Also, you have a plural argument `relationships` that actually accepts a singular object??? Very confusing function signature